docs/ENVIRONMENT.md

```diff
@@ -714,6 +714,9 @@ Tools using the Varlink protocol (such as `varlinkctl`) or sd-bus (such as
   no effect if the background color is explicitly selected via the relevant
   `--background=` switch of the tool.
 
+* `$SYSTEMD_SET_TERM_TITLE` – Takes a boolean. When false the terminal title
```

Comment:
I am not a big fan of needless abbreviations, in particular for options that aren't first class stuff.

Hence maybe: SYSTEMD_ADJUST_TERMINAL_TITLE? Yeah, it's quite long, but I think it's clearer, dunno?

The thing is that we do more than just "set" the window title, we prepend things to it, as the window title ANSI sequences flow through our pty, hence I think "adjust" is a bit more precise.

Reply:
the ansi sequence specs typically call the concept "window title" btw. Hence I think the prose version should use that wording, i.e. "When false the terminal window title…"

Reply:
Sure, I can rename it.